### PR TITLE
Update docs to reflect nginx support

### PR DESCRIPTION
**Pull request by diracdeltas**

### CODE REVIEW

File: letsencrypt/client/plugins/nginx/parser.py [hunk 1]
```diff
@@ -41,9 +41,6 @@ def _parse_recursively(self, filepath):
         directives inside 'http' and 'server' blocks. Note that this only
         reads Nginx files that potentially declare a virtual host.
 
-        .. todo:: Can Nginx 'virtual hosts' be defined somewhere other than in
-        the server context?
-
```
Comment (diracdeltas) [inline letsencrypt/client/plugins/nginx/parser.py]: Deleted because I am fairly certain they can't be.

File: README.rst [hunk 2]
```diff
@@ -21,7 +21,7 @@ All you need to do is::
 
   user@www:~$ sudo letsencrypt -d www.example.org auth
 
-and if you have a compatbile web server (Apache), Let's Encrypt can
+and if you have a compatbile web server (Apache or Nginx), Let's Encrypt can
```
Comment (kuba) [inline README.rst]: Yay! :)

File: letsencrypt/client/plugins/nginx/configurator.py [hunk 3]
```diff
@@ -554,6 +552,10 @@ def cleanup(self, achalls):
 def nginx_restart(nginx_ctl):
     """Restarts the Nginx Server.
 
+    .. todo:: Nginx restart is fatal if the configuration references
+        non-existent SSL cert/key files. Remove references to /etc/letsencrypt
+        before restart.
```
Comment (kuba) [inline letsencrypt/client/plugins/nginx/configurator.py]: great

File: letsencrypt/client/plugins/nginx/configurator.py [hunk 4]
```diff
@@ -270,7 +268,7 @@ def _make_server_ssl(self, vhost):
         a 'listen 443 ssl' directive to the server block.
 
         .. todo:: Maybe this should create a new block instead of modifying
-        the existing one?
+            the existing one?
```
Comment (kuba) [inline letsencrypt/client/plugins/nginx/configurator.py]: good catch

**Comment by kuba** [comment]
LGTM :+1: . Thanks once again for the nginx plugin, great job! :)

**Comment by diracdeltas** [comment]
thanks for the reviews!

**Pull request merged**